### DISCUSSION TITLE
Resettare accesso admin senza recovery key

### DISCUSSION THREAD

**Discussion by Cangio84** [globaleaks/globaleaks-whistleblowing-software#3869] [Q&A]
Buongiorno,
per sbaglio è stato effettuato un reset della password di admin della piattaforma senza aver salvato la recovery key.
è possibile tramite ssh recuperare l'accesso? qui le credenziali sono ancora attive
Grazie,

**Comment by evilaliv3** [comment]
Buongiorno,

Il sistema è attivo in produzione? Ci sono segnalazioni a sistema?

Possiamo fornirvi eventuali istruzioni per ri-accedere come amministrazioni al sistema ed estrarvi le configurazioni ma nel caso di assenza delle chiavi di recovery è suggerito effettuare un nuovo setup in quanto la chiave di recovery degli amministratori è l'unica garanzia di assenza di perdita dati di segnalazioni nel caso in cui i riceventi perdano i propri accessi.

Eventualmente può raggiungermi su community.globaleaks.org

**Reply by Cangio84** [reply]
Gentile evilaliv3,
grazie per la pronta risposta.
si a sistema sono presenti delle segnalazioni e non abbiamo le recovery key.
buono a sapersi che comunque esista la possibilità di recuperare gli accessi.
valuteremo il da farsi e nel caso procederemo con un nuovo setup.
grazie
